components/patient-access-rules/api/pom.xml

```diff
@@ -30,7 +30,7 @@
 
   <properties>
     <checkstyle.skip>true</checkstyle.skip>
-    <coverage.instructionRatio>0.69</coverage.instructionRatio>
+    <coverage.instructionRatio>0.67</coverage.instructionRatio>
```

Comment:
It's really frowned upon to decrease the coverage. If there's new code, it should have new tests.

Anyway, all this code shouldn't be moved into its own module, in `components/record-locking`, complete with `api`, `impl`, `ui`, `pageobjects` and `tests` submodules.